models.go

```diff
@@ -136,6 +134,40 @@ type respUserGet struct {
 	// TODO: extattr external_profile external_position
 }
 
+// respUserGet 读取成员响应
+type respUserGet struct {
+	respCommon
+
+	UserDetail
+}
+
+// reqUserByDept 部门成员请求
+type reqUsersByDept struct {
+	DeptID     int64 `json:"department_id"`
+	FetchChild bool  `json:"fetch_child"`
```

Comment:
You can remove the `json` tags, as they're unused by the client. Request parameters are constructed with the `intoURLValues` method.